plugins/components/conversionlayer.go

```diff
@@ -378,11 +378,21 @@ func getActionFunc(cmd Command) cli.ActionFunc {
 }
 
 func ConvertContext(baseContext *cli.Context, flagsToConvert ...Flag) (*Context, error) {
+	print(baseContext)
 	pluginContext := &Context{
 		CommandName:      baseContext.Command.Name,
 		Arguments:        baseContext.Args(),
 		PrintCommandHelp: getPrintCommandHelpFunc(baseContext),
 	}
+
+	if baseContext.Parent() != nil {
```

Comment:
Please explain what this is used for, couldn't understand from the code and couldn't find a usage in other PRs